units/stats_amboy.go

```diff
@@ -90,4 +91,24 @@ func (j *amboyStatsCollector) Run(ctx context.Context) {
 			"stats":   remoteQueue.Stats(ctx),
 		})
 	}
+
+	remoteQueueGroup := j.env.RemoteQueueGroup()
+	if !j.ExcludeRemote && remoteQueueGroup != nil {
+		// Log queue stats for any queue in the queue group that still has jobs
+		// to process or has only finished processing all of its jobs very
+		// recently.
+		for _, queueName := range remoteQueueGroup.Queues(ctx) {
+			queue, err := remoteQueueGroup.Get(ctx, queueName)
+			if err != nil {
+				j.AddError(errors.Wrapf(err, "getting queue stats for queue '%s' in queue group", queueName))
+				continue
+			}
+
+			j.logger.Info(message.Fields{
```

Comment:
How often will this log? Should we be concerned about it potentially exhausting our splunk limit?
I wonder if there is maybe a way the logging to long queues here instead of limiting the splunk alert itself?

Reply:
This logs once a minute, once per queue. My expectation is that this will create a total of 5-10 times per minute, so this doesn't pose a risk to our Splunk rate limit.